back-end/www/models/model_operations/location_operations.py

```diff
@@ -0,0 +1,193 @@
+"""Functions to operate the location table."""
+
+from models.model import db
+from models.model import Location
+
+def create_location(factory_id, photo_year=1997, photo_url=""):
```

Comment:
I would say do not five default value to year. If it has no year, an error should be returned.

Reply:
also, bounding box should be provided as the input

Reply:
Move to Answer table.